server/routes/reporting/standard-report/standard-report-print.js

```diff
@@ -292,6 +304,70 @@ async function standardReportPrint(app, req, res, reportKey, data) {
   }
 };
 
+function sortTableData({ sortBy, sortDirection }, tableData, reportData) {
+  if (reportData.bespokeReport) return; // we do not sort bespoke reports for now
```

Comment:
Maybe change this to;

`if (reportData.bespokeReport.body) return;`

I have tested each of the reports that are bespoke but dont contain a seperate body file, and these all sort and work as expected (other than trial-attendance, which is in another comment). 
Would mean reports such as persons attending can be sorted however they please for scanning and checking in jurors